src/components/main-form.js

```diff
@@ -209,7 +213,34 @@ class MainForm extends React.Component {
             onChange={this.updateInput}
           />
         </div>
-
+        <div className='row'>
+          <div className='col font-weight-bold'>Month</div>
+          <div className='col font-weight-bold'>{this.state.shorterOption.term} Yr Mortgage Payment</div>
+          <div className='col font-weight-bold'>{this.state.shorterOption.term} Yr Investment Payment</div>
```

Comment:
You could really use reacstrap directly here and make this a lot cleaner and explicit: https://reactstrap.github.io/components/layout/

For the table's header, I would create a styled component named `ColHeader` like so:

const ColHeader = styled(Col)`
  font-weight: bold;
`

Less CSS floating around, more composition, and much more explicit (less divs and more Cols, ColHeaders and Rows)

Reply:
Where do I get `Col`? Is it possible to extend from Bootstrap to get `col`?

Reply:
See link in my first comment:

`import { Container, Row, Col } from 'reactstrap';`

We're already using reactstrap for our `Input` component. There is almost never a reason to use Bootstrap directly if we're using reactstrap.